src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeProcessorValidationTest.java

```diff
@@ -15,20 +19,27 @@
 import java.util.List;
 
 import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.Mockito.when;
 import static uk.gov.hmcts.reform.bulkscanprocessor.helper.InputEnvelopeCreator.inputEnvelope;
 import static uk.gov.hmcts.reform.bulkscanprocessor.helper.InputEnvelopeCreator.scannableItem;
 
 @SuppressWarnings("checkstyle:LineLength")
+@RunWith(MockitoJUnitRunner.class)
 public class EnvelopeProcessorValidationTest {
 
+    @Mock
+    private ContainerMappings containerMappings;
```

Comment:
not needed, you can just use `List<Mapping>` in the tests. Up to you.

Reply:
@rkondratowicz removed the above and using the list now